DEHPCommon/UserInterfaces/ViewModels/Interfaces/IObjectBrowserViewModel.cs

```diff
@@ -35,72 +35,7 @@ namespace DEHPCommon.UserInterfaces.ViewModels.Interfaces
     /// <summary>
     /// Interface definition for the <see cref="ObjectBrowserViewModel"/>
     /// </summary>
-    public interface IObjectBrowserViewModel
+    public interface IObjectBrowserViewModel : IObjectBrowserBaseViewModel
```

Comment:
Is there still a need for this interface?

Reply:
Note that `IObjectBrowserBaseViewModel `contains a Dispoable() and Sonarcloud recommends the use of `IDisposable`

Reply:
yes and no since its empty, but this way there is a IObjectBrowserViewModel and INetCHangePreviewVIewModel in the container, but I could also register the ObjectBrowserViewModel as IObjectBrowserBaseViewModel, so it's just naming preference . What do you think is preferable @alexatrhea?